django/contrib/admin/tests.py

```diff
@@ -2,10 +2,21 @@
 from unittest import SkipTest
 
 from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+from django.test import modify_settings
 from django.utils.module_loading import import_string
 from django.utils.translation import ugettext as _
 
 
+class CSPMiddleware(object):
+
+    def process_response(self, request, response):
+        response['Content-Security-Policy'] = "default-src 'self'"
+        return response
+
+
+@modify_settings(
+    MIDDLEWARE_CLASSES={'append': 'django.contrib.admin.tests.CSPMiddleware'}
```

Comment:
add trailing comma (if more items are added later, we won't have to modify this line again which makes git blame a bit cleaner)